packages/@react-spectrum/illustrated-message/stories/IllustratedMessage.stories.tsx

```diff
@@ -29,11 +31,21 @@ storiesOf('IllustratedMessage', module)
   )
   .add(
     'heading, description',
-    () => render({heading: 'No Results', description: 'Try another search', illustration: noResultsImg(), ariaLevel: 3})
+    () => render({heading: 'No Results', description: 'Try another search', illustration: noResultsImg()})
   );
 
 function render(props: any = {}) {
+  let {
+    illustration,
+    heading,
+    description,
+    ...otherProps
+  } = props;
   return (
-    <IllustratedMessage {...props} />
+    <IllustratedMessage {...otherProps}>
+      {description && <Content>{description}</Content>}
+      {heading && <Heading>{heading}</Heading>}
+      {illustration}
```

Comment:
You need to add `aria-hidden="true"` to the svg, unless there's a heading or description, in which case you need an `aria-label`. Maybe add a parameter to the `noResultsImg` function to pass props to it?